calculator.js

```diff
@@ -1,2 +1,76 @@
 var calculator = {
+
+  value: 0,
+
+  add: function(x, y) {
+    if(y) {
+      var sum = x + y
```

Comment:
Inside a function, a variable should only be declared once with `var` or JS will complain. Preferably we should do this at the top.

Here's an example:
```js
function(x, y) {
    var sum;
    if (y) {
        sum = x + y
    } else {
        sum = this.value + x;
    }
    this.value = sum
    return sum
}
```